db/migrate/20220920231535_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Hello and excellent work on the data model, but please as the ERD file indicate that the `Posts, Comments and likes` have an `Author` **and are referenced in the foreign_key to the table of users**